dmutils/content_loader.py

```diff
@@ -1,32 +1,54 @@
+import collections
 import yaml
 import inflection
 import re
 import os
 
+from .config import convert_to_boolean, convert_to_number
+
 
 class ContentBuilder(object):
+    """An ordered set of sections each made up of one or more questions.
+
+    Usage::
+
+        >>> content = ContentBuilder(sections)
```

Comment:
Ah, bums. Thanks though, this also highlighted I was referencing the wrong method in some of the docstrings.